`load-from-h2` fails on `master` with Command failed with exception: ERROR Clearing default entries created by Liquibase migrations...

### Describe the bug

`load-from-h2` doesn't work on `master` regardless of version used to `dump-to-h2`

### To Reproduce

1. Spin up fresh instance on `master` on H2 database. Create one question or something.
2. Spin up fresh instance on `master` on Postgres database.
4. java -jar metabase.jar load-from-h2 path-to-h2-db
5. See error


### Expected behavior

`load-from-h2` should work

### Logs

```
2023-10-09 11:51:43,545 ERROR cmd.copy :: [FAIL]

clojure.lang.ExceptionInfo: ERROR Clearing default entries created by Liquibase migrations...: Batch entry 1 TRUNCATE TABLE ConnectionImpersonation CASCADE was aborted: ERROR: relation "connectionimpersonation" does not exist  Call getNextException to see other errors in the batch. {}
	at metabase.cmd.copy$do_step$fn__106500.invoke(copy.clj:37)
	at metabase.cmd.copy$do_step.invokeStatic(copy.clj:33)
	at metabase.cmd.copy$do_step.invoke(copy.clj:31)
	at metabase.cmd.copy$fn__106690$copy_BANG___106695$fn__106696.invoke(copy.clj:403)
	at metabase.cmd.copy$fn__106690$copy_BANG___106695.invoke(copy.clj:379)
	at metabase.cmd.load_from_h2$load_from_h2_BANG_.invokeStatic(load_from_h2.clj:36)
	at metabase.cmd.load_from_h2$load_from_h2_BANG_.invoke(load_from_h2.clj:26)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.cmd$load_from_h2.invokeStatic(cmd.clj:74)
	at metabase.cmd$load_from_h2.invoke(cmd.clj:68)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at metabase.cmd$run_cmd$fn__107821.invoke(cmd.clj:294)
	at metabase.cmd$run_cmd.invokeStatic(cmd.clj:293)
	at metabase.cmd$run_cmd.invoke(cmd.clj:283)
	at clojure.lang.Var.invoke(Var.java:388)
	at metabase.core$run_cmd.invokeStatic(core.clj:178)
	at metabase.core$run_cmd.invoke(core.clj:176)
	at metabase.core$entrypoint.invokeStatic(core.clj:200)
	at metabase.core$entrypoint.doInvoke(core.clj:195)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
	at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at metabase.bootstrap.main(Unknown Source)
Caused by: java.sql.BatchUpdateException: Batch entry 1 TRUNCATE TABLE ConnectionImpersonation CASCADE was aborted: ERROR: relation "connectionimpersonation" does not exist  Call getNextException to see other errors in the batch.
	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:165)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2402)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:574)
	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:896)
	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:919)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeBatch(NewProxyStatement.java:1013)
	at metabase.cmd.copy$clear_existing_rows_BANG_$fn__106622.invoke(copy.clj:304)
	at metabase.cmd.copy$do_with_disabled_db_constraints.invokeStatic(copy.clj:272)
	at metabase.cmd.copy$do_with_disabled_db_constraints.invoke(copy.clj:268)
	at metabase.cmd.copy$clear_existing_rows_BANG_.invokeStatic(copy.clj:290)
	at metabase.cmd.copy$clear_existing_rows_BANG_.invoke(copy.clj:283)
	at metabase.cmd.copy$fn__106690$copy_BANG___106695$fn__106696$fn__106707.invoke(copy.clj:404)
	at metabase.cmd.copy$do_step$fn__106500.invoke(copy.clj:34)
	... 29 more
Caused by: org.postgresql.util.PSQLException: ERROR: relation "connectionimpersonation" does not exist
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2713)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2401)
	... 40 more
Command failed with exception: ERROR Clearing default entries created by Liquibase migrations...: Batch entry 1 TRUNCATE TABLE ConnectionImpersonation CASCADE was aborted: ERROR: relation "connectionimpersonation" does not exist  Call getNextException to see other errors in the batch.
```

### Information about your Metabase installation

```JSON
master
```


### Severity

P1

### Additional context

_No response_

--- a/src/metabase/cmd/copy.clj
+++ b/src/metabase/cmd/copy.clj
@@ -4,10 +4,7 @@
   data from an application database to any empty application database for all combinations of supported application
   database types."
   (:require
-   [clojure.java.classpath :as classpath]
    [clojure.java.jdbc :as jdbc]
-   [clojure.string :as str]
-   [clojure.tools.namespace.find :as ns.find]
    [honey.sql :as sql]
    [metabase.config :as config]
    [metabase.db.connection :as mdb.connection]
@@ -375,7 +372,6 @@
                                         table-name table-name)]]
         (jdbc/execute! target-db-conn sql)))))
 
-
 (s/defn copy!
   "Copy data from a source application database into an empty destination application database."
   [source-db-type     :- (s/enum :h2 :postgres :mysql)
@@ -383,9 +379,7 @@
    target-db-type     :- (s/enum :h2 :postgres :mysql)
    target-data-source :- javax.sql.DataSource]
   ;; make sure the entire system is loaded before running this test, to make sure we account for all the models.
-  (doseq [ns-symb (ns.find/find-namespaces (classpath/system-classpath))
-          :when   (and (str/starts-with? ns-symb "metabase")
-                       (not (str/includes? ns-symb "test")))]
+  (doseq [ns-symb u/metabase-namespace-symbols]
     (classloader/require ns-symb))
   ;; make sure the source database is up-do-date
   (step (trs "Set up {0} source database and run migrations..." (name source-db-type))

--- a/src/metabase/util/jvm.clj
+++ b/src/metabase/util/jvm.clj
@@ -268,11 +268,11 @@
 ;; This is made `^:const` so it will get calculated when the uberjar is compiled. `find-namespaces` won't work if
 ;; source is excluded; either way this takes a few seconds, so doing it at compile time speeds up launch as well.
 (defonce ^:const ^{:doc "Vector of symbols of all Metabase namespaces, excluding test namespaces. This is intended
-                        for use by various routines that load related namespaces, such as task and events
-                        initialization."}
- metabase-namespace-symbols
+  for use by various routines that load related namespaces, such as task and events
+  initialization."}
+  metabase-namespace-symbols
   (vec (sort (for [ns-symb (ns.find/find-namespaces (classpath/system-classpath))
-                   :when   (and (str/starts-with? ns-symb "metabase.")
+                   :when   (and (str/starts-with? ns-symb "metabase")
                                 (not (str/includes? ns-symb "test")))]
                ns-symb))))
 

